### PR TITLE
Contact-page

### PR DESCRIPTION
**Pull request by judytabaca**

### CODE REVIEW

File: src/pages/ContactPage.vue [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
+          Monterail Sp. z o.o. <br />
+          ul. Oławska 27-29 <br />
+          50-123 Wrocław <br />
+          Poland
+        </p>
+        <img src="@/assets/img/emailicon.svg" alt="email icon" />
+        <p>hello@monterail.com</p>
+        <img src="@/assets/img/phoneicon.svg" alt="phone icon" />
+        <p>+48 533 600 136</p>
+      </div>
```
Comment (szymonlicau) [inline src/pages/ContactPage.vue]: I think this part is duplicated in `ContactUs` component.
We could try to refactor this so we could re-use this part in both places - what do you think?

Comment (judytabaca) [reply src/pages/ContactPage.vue]: Valid point taken :D Thank you 🥇

File: src/pages/ContactPage.vue [hunk 2]
```diff
@@ -63,20 +66,22 @@ h1 {
...
+    padding: 0;
     padding-left: 4em;
     font-size: 18px;
     font-weight: 400;
     font-family: "Roboto Mono";
     line-height: 32px;
     letter-spacing: 0.015em;
     color: $tuna;
+    border-radius: 0;
   }
```
Comment (szymonlicau) [inline src/pages/ContactPage.vue]: Do we need this? I think most of those styles are already applied by the `AddressCard` component 🤔
If it's just some small modifications we should leave it at only those and remove the duplications 😄

**Review by szymonlicau — CHANGES_REQUESTED** [review]

File: src/pages/ContactPage.vue (hunk 2 above)
Comment (judytabaca) [reply src/pages/ContactPage.vue]: Thank you for keeping me on the right track 👼🏽

**Review by szymonlicau — APPROVED** [review]

**Pull request merged**